code-challenges.test.js

```diff
@@ -15,35 +15,142 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+
+describe('newArray', () => {
+  const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+  const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+  it('takes in an array, removes the first item from the array and shuffles the remaining content', () => {
+    expect(newArray(colors1)).toEqual(expect.arrayContaining(colors1));
+    expect(newArray(colors2)).toEqual(expect.arrayContaining(colors2));
+  });
+});
+
+// ● newArray › takes in an array, removes the first item from the array and shuffles the remaining content
+//
+//    ReferenceError: newArray is not defined
+
+//const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+//const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
 
 // b) Create the function that makes the test pass.
-
+// create a function
+// use shift to remove the first item in an array
+// use sort to shuffle the array after removing the first item
+// look up different ways to use sort to shuffle the array
+
+const newArray = (array) => {
+  array.shift()
+  array.sort((a, b) => 0.6 - Math.random())
+  return array
+}
```

Comment:
Awesome work! Why did you use .6 instead of .5? Just wondering.

Reply:
.6 came from me playing with the code to get a better understanding on how it worked.